src/layouts/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import Helmet from 'react-helmet';
-import './index.scss'
-import '../components/css/custom.scss';
+import '../assets/styles/index.scss'
```

Comment:
we can follow below import order
1. Api Dependencies
2. Component Dependencies
3. CSS, assets